src/components/global/localNav/localNav.ts

```diff
@@ -157,6 +168,10 @@ export class LocalNav extends LitElement {
       link._navExpanded = this._expanded || this.pinned;
       link._navExpandedMobile = this._mobileExpanded;
     });
+
+    this._dividers.forEach((divider: any) => {
+      divider._navExpanded = this._expanded || this.pinned;
```

Comment:
could add an `on-expand` or `on-pin` action to expand on `on-toggle`

Reply:
`on-toggle` just emits the new pinned state value. An event for the expand/collapse on mouse in/out probably doesn't have a use case that I can think of at least.